### DISCUSSION TITLE
Geemap leafmap installation

### DISCUSSION THREAD

**Discussion by riteta-rimon** [giswqs/geog-312#117] [Q&A] [answered]
Hello Dr.Wu, 

I followed the instruction on geemap leafmap installation and this error always come up. 

![Screenshot 2024-10-16 102720](https://github.com/user-attachments/assets/2382795c-1c34-415f-8861-128a791b16b6)

Thank you

**Comment by giswqs** [comment, marked as answer]
If mamba does not work for your, try `pip install geemap leafmap`

**Reply by riteta-rimon** [reply]
Thank you Sir
